src/components/datatable/DocumentsTable.vue

```diff
@@ -123,47 +187,47 @@
           this.columnDefs = [
             getColDef(this, fields.title, { cellRendererFramework: DocumentLink, width: 300 }),
             getColDef(this, fields.activities, { cellRendererFramework: DocumentActivities, width: 300 }),
-            getColDef(this, fields.categories, { width: 120 }),
+            getColDef(this, fields.categories, { width: 120, sortable: false }),
             getColDef(this, fields.article_type, { width: 120 }),
-            { cellRendererFramework: MarkerQuality, width: 30 }
+            getColDef(this, fields.quality, { cellRendererFramework: MarkerQuality, width: 30, comparator: qualComp })
           ];
         }
 
         if (this.documentType === 'book') {
           this.columnDefs = [
             getColDef(this, fields.title, { cellRendererFramework: DocumentLink }),
             getColDef(this, fields.book_types),
-            getColDef(this, fields.author),
+            getColDef(this, fields.author, { sortable: false }),
```

Comment:
I think it would be better to modify `/src/js/constants/fieldsProperties.json` and add a property `sortable`. As most of fields are sortable (is it true ? if no, inversed logic), if it's not defined, it means it's sortable. If it's defined and `True`, it's sortable. Then `getColDef()` will look like something : 

```js

  function getColDef(vm, field, options) {
    options = options || {};
    const result = {
      headerName: vm.$gettext(field.name),
      field: field.name,
      _fieldDefinition: field,
      cellRendererFramework: options.cellRendererFramework || DocumentField,
      resizable: true,
      sortable: field.sortable === undefined || field.sortable === True
      // if most of fields are NOT sortable, then undefined means False: 
      // sortable: field.sortable === True
    };
    return Object.assign(result, options);
  }
```